docs/proton_c_conversion.md

```diff
@@ -31,3 +37,54 @@ These are defaults based on what has been implemented for ARM boards.
 | [Backlight](feature_backlight.md)   | Forces [task driven PWM](feature_backlight.md#software-pwm-driver) until ARM can provide automatic configuration |
 | USB Host (e.g. USB-USB converter)   | Not supported (USB host code is AVR specific and is not currently supported on ARM)                              |
 | [Split keyboards](feature_split_keyboard.md) | Not supported yet                                                                                       |
+
+## Manual Conversion
+
+To use the Proton C natively, without having to specify `CTPC=yes`, you need to change the `MCU` line in `rules.mk`:
+
+```
+MCU = STM32F303
+```
+
+Remove these variables if they exist:
+
+* `BOOTLOADER`
+* `EXTRA_FLAGS`
+
+Finally convert all pin assignments in `config.h` to the stm32 equivalents.
+
+| Pro Micro | Proton C | | Proton C | Pro Micro |
+|-----------|----------|-|----------|-----------|
+| `D3` | `A9` | | 5v | RAW (5v) |
+| `D2` | `A10` | | GND | GND |
+| GND | GND | | FLASH | RESET |
+| GND | GND | | 3.3v | VCC <sup>1</sup> |
+| `D1` | `B7` | | `A2` | `F4` |
+| `D0` | `B6` | | `A1` | `F5` |
+| `D4` | `B5` | | `A0` | `F6` |
+| `C6` | `B4` | | `B8` | `F7` |
+| `D7` | `B3` | | `B13` | `B1` |
+| `E6` | `B2` | | `B14` | `B3` |
+| `B4` | `B1` | | `B15` | `B2` |
+| `B5` | `B0` | | `B9` | `B6` |
+| `B0` (RX LED) | `C13` <sup>2</sup> | | `C13` <sup>2</sup> | `D5` (TX LED) |
+
+You can also make use of several new pins on the extended portion of the Proton C:
+
+| Left | | Right | Comments |
+|------|-|-------|----------|
+| `A4`<sup>3</sup> | | `B10` |
+| `A5`<sup>4</sup> | | `B11` |
+| `A6` | | `B12` |
+| `A7` | | `A14`<sup>5</sup> (SWCLK) |
+| `A8` | | `A13`<sup>5</sup> (SWDIO) |
+| `A15` | | RESET<sup>6</sup> |
```

Comment:
Actually, I don't think that you can actually use these, as they're not defined if CTPC is enabled. 

Yeah, it's not: 
https://github.com/qmk/qmk_firmware/blob/dc79792ab463f614e3aba979dace7d9625b340bf/quantum/config_common.h#L146-L183

Reply:
I'm not sure I follow here. This section is about converting so you don't use CTPC. You should be able to use all the pins once you've converted.

Reply:
You can use all the Pro Micro pins, yes.  

But, as it is, the pins on the rest of the Proton C are NOT defined (the break away part), so they can't be used via the short names (A4, A5, A6, etc).  You'd need to use `PAL_LINE(GPIOA, 6)` instead of A6.

Reply:
Sure, but this section is specifically about using the Proton C natively, without using CTPC. None of what you're saying applies here.